### PR TITLE
feat(draw_buf): add LV_STRIDE_AUTO to let lvgl calculate stride

### PR DESCRIPTION
**Pull request by XuNeo**
Help us review this PR! Anyone can [approve it or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### Description of the feature or fix

Add macro for automatically stride calculation.

> On a side note, what do you think about having #define LV_STRIDE_AUTO 0?
> https://github.com/lvgl/lvgl/issues/5261

### Checkpoints
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html)

### CODE REVIEW

**Review by PGNetHun — APPROVED** [review]

File: src/draw/lv_draw_buf.h [hunk 1]
```diff
@@ -21,6 +21,9 @@ extern "C" {
  *      DEFINES
  *********************/
 
+/*Use this value to let LVGL calculate stride automatically*/
+#define LV_STRIDE_AUTO 0
+
```
Comment (kisvegabor) [inline src/draw/lv_draw_buf.h:27]: To make available in MicroPython too.
```suggestion
LV_EXPORT_CONST_INT(LV_STRIDE_AUTO)
```

cc @PGNetHun

**Review by kisvegabor — CHANGES_REQUESTED** [review]

File: src/draw/lv_draw_buf.h (hunk 1 above)
Comment (kisvegabor) [reply src/draw/lv_draw_buf.h:27]: It needs a `;` too.

Comment (XuNeo) [reply src/draw/lv_draw_buf.h:27]: Ah, get it. Thanks. Fixed.

**Review by FASTSHIFT — APPROVED** [review]

**Comment by PGNetHun** [comment]
@kisvegabor 
Could you please merge this PR to `master`?
After this I want to update [MicroPython PR](https://github.com/lvgl/lv_binding_micropython/pull/311)
Thank you!

**Review by kisvegabor — APPROVED** [review]

**Pull request merged**